onnx/defs/math/defs.cc

```diff
@@ -1046,4 +1046,27 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input and output types to float tensors.")
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* Expand_ver8_doc = R"DOC(
+Expand the input tensor following the given shape.
```

Comment:
This description is too ambiguous. User might think the `given shape` will be exactly the `output shape` (which is not).

Reply:
Agree with @bddppq. This operator is not very popular (For example, in tensorflow and numpy, people use expand_dim + tile instead). I think that only with the vague description users cannot get the precise definition of the op. Please put more details and make sure the backend developers will know how to implement it after reading our description.